packages/@aws-cdk/custom-resource-handlers/lib/aws-s3-deployment/bucket-deployment-handler/index.py

```diff
@@ -108,7 +109,7 @@ def cfn_error(message=None):
             physical_id = "aws.cdk.s3deployment.%s" % str(uuid4())
         else:
             if not physical_id:
-                cfn_error("invalid request: request type is '%s' but 'PhysicalResourceId' is not defined" % {request_type})
+                cfn_error("invalid request: request type is '%s' but 'PhysicalResourceId' is not defined" % request_type)
```

Comment:
Just for my own understanding, is this to fix [CWE-93](https://cwe.mitre.org/data/definitions/93.html)? If so, how does this actually fix it?

Reply:
The fix is at line 34: using `encode()` to sanitize the logged error message.

Reply:
Oh I was thinking that was only to fix CWE-117. Does using `encode()` fix both vulnerabilities then?

Reply:
It should fix both (CWE-117 and CWE-93 are related vulnerabilities), as suggested [here](https://community.veracode.com/s/article/How-to-Fix-CWE-117-Improper-Output-Neutralization-for-Logs).
Not sure if AWS Inspector requires an implementation with `urllib.parse.quote`.